lib/merchant_repository.rb

```diff
@@ -1,7 +1,7 @@
 require 'bigdecimal'
 require 'bigdecimal/util'
 require_relative 'merchant'
-require_relative 'file_loader'
+require_relative 'load_file'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.